ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/README.md

```diff
@@ -34,7 +34,7 @@ The following properties are written to JCR for this Form Text component and are
 1. `./jcr:title` - defines the label to use for this field
 2. `./hideTitle` - if set to `true`, the label of this field will be hidden
 3. `./name` - defines the name of the field, which will be submitted with the form data
-4. `./default` - defines the default value of the field
+4. `./default` - defines the default text of the field
```

Comment:
default text is confusing/ambiguous - as it can be label to - value sounds better